elements/health.lua

```diff
@@ -189,38 +186,16 @@ local function ForceUpdate(element)
 	return Path(element.__owner, 'ForceUpdate', element.__owner.unit)
 end
 
---[[ Health:SetFrequentUpdates(state)
-Used to toggle frequent updates.
-
-* self  - the Health element
-* state - the desired state of frequent updates (boolean)
---]]
-local function SetFrequentUpdates(element, state)
-	if(element.frequentUpdates ~= state) then
-		element.frequentUpdates = state
-		if(element.frequentUpdates) then
-			element.__owner:UnregisterEvent('UNIT_HEALTH', Path)
-			element.__owner:RegisterEvent('UNIT_HEALTH_FREQUENT', Path)
-		else
-			element.__owner:UnregisterEvent('UNIT_HEALTH_FREQUENT', Path)
-			element.__owner:RegisterEvent('UNIT_HEALTH', Path)
-		end
-	end
-end
+local function noop() end
```

Comment:
`nop` is a global function provided by blizz.

Reply:
orly? O_o

Reply:
https://www.townlong-yak.com/framexml/live/UIParent.lua#5188

Reply:
🤔 👌